java-client/src/main/java/org/scassandra/http/client/PreparedStatementPreparation.java

```diff
@@ -0,0 +1,59 @@
+package org.scassandra.http.client;
+
+public final class PreparedStatementPreparation {
```

Comment:
Could you add an EqualsVerifier test? Like for BatchExecution as people will reply on the equals method in their tests.